webplugin/js/app/kommunicate.js

```diff
@@ -116,6 +116,45 @@ $applozic.extend(true, Kommunicate, {
             }, SET_TIMEOUT_DURATION);
         }
     },
+    updateConversationMetadata: function (conversationMetadata) {
+        if (
+            conversationMetadata != null &&
+            typeof conversationMetadata !== 'undefined'
+        ) {
+            var metadataToSend = Object.assign({}, conversationMetadata);
+            var metadataToShow = Object.assign({}, conversationMetadata.metadata);
+            if (
+                kommunicateCommons.isObject(metadataToSend) &&
+                kommunicateCommons.isObject(metadataToShow) &&
+                metadataToSend.groupId &&
+                metadataToSend.metadata
+            ) {
```

Comment:
can we directly unpack group id and metadata and then directly use that?
this code is a little confusing.